dlp/README.md

```diff
@@ -0,0 +1,29 @@
+# DLP Sample
```

Comment:
Could you follow: https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/iot/README.md